wagtail/admin/templates/wagtailadmin/admin_base.html

```diff
@@ -29,6 +29,7 @@
 
             wagtailConfig.STRINGS = {
                 EDIT: "{% trans 'Edit' %}",
+                DELETE: "{% trans 'Delete' %}",
                 PAGE: "{% trans 'Page' %}",
                 PAGES: "{% trans 'Pages' %}",
                 LOADING: "{% trans 'Loading...' %}",
```

Comment:
Could be worth changing these three dots to an ellipsis `…` like below.

Reply:
I was wondering about this as well – is the hassle of re-doing the translations worth it?

Reply:
https://github.com/wagtail/wagtail/pull/4218